tasks/postgres-standby-barman.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: Compute barman remote URL
+  set_fact:
+    postgres_barman_remote_url: "{{ lookup('template', 'barman_remote_url.j2') | trim }}"
```

Comment:
I feel that 'templates' in an Ansible role should contain mostly configuration template files. This template is here only to help construct this `fact`.

What do you think about using ansible's `vars:` instead?

Something like:

```
  set_fact:
    postgres_barman_remote_url: "{{ scheme }}{{ user }}{{ server }}{{ delimiter }}{{ path }}"
  vars:
    scheme: {{ postgres_barman_rsync_enabled | ternary('rsync://', '') }}
    user: {{ postgres_barman_remote_user is defined | ternary(postgres_barman_remote_user + '@', '') }}
    server: {{ postgres_barman_server }}
    delimiter: {{ not postgres_barman_rsync_enabled | ternary(':', '')}}
    path: {{ postgres_barman_path_prefix }}
```

I think it helps readability. WDYT?

Reply:
It definitely helps, thanks a lot @paulRbr !

I have slightly adapted your patch so we don't use `not … | ternary(…)` but `… | ternary(…)` and added the password file path option by default when using rsync, as we create it in the role.